frontend/lib/api/labels/index.ts

```diff
@@ -0,0 +1,21 @@
+import { PollingStationType } from "@kiesraad/api";
+import { t } from "@kiesraad/i18n";
+
+export const labelForPollingStationType = withEntries<{ [K in PollingStationType]: string }>({
+  FixedLocation: t("polling_station.type.FixedLocation"),
+  Special: t("polling_station.type.Special"),
+  Mobile: t("polling_station.type.Mobile"),
+});
+
+function withEntries<T extends { [K in keyof T]: T[K] }>(
```

Comment:
I would prefer using the standard `Object.entries()` directly (less surprises reading the code), but I think that this might not be needed at all anymore.

Reply:
removed, we do need to share this somewhere I think. It's used in 3 places currently.